src/platform/packages/shared/response-ops/recurring-schedule-form/components/recurring_schedule_form_fields.tsx

```diff
@@ -39,30 +40,26 @@ import { parseSchedule } from '../utils/parse_schedule';
 import { getPresets } from '../utils/get_presets';
 import { getWeekdayInfo } from '../utils/get_weekday_info';
 import { RecurringSchedule } from '../types';
-import {
-  RECURRING_SCHEDULE_FORM_FREQUENCY_DAILY,
-  RECURRING_SCHEDULE_FORM_FREQUENCY_WEEKLY_ON,
-  RECURRING_SCHEDULE_FORM_FREQUENCY_NTH_WEEKDAY,
-  RECURRING_SCHEDULE_FORM_FREQUENCY_YEARLY_ON,
-  RECURRING_SCHEDULE_FORM_FREQUENCY_CUSTOM,
-  RECURRING_SCHEDULE_FORM_TIMEZONE,
-  RECURRING_SCHEDULE_FORM_COUNT_AFTER,
-  RECURRING_SCHEDULE_FORM_COUNT_OCCURRENCE,
-  RECURRING_SCHEDULE_FORM_RECURRING_SUMMARY_PREFIX,
-} from '../translations';
+import * as i18n from '../translations';
 
 /**
  * Using EuiForm in `div` mode since this is meant to be integrated in a larger form
  */
 const UseField = getUseField({ component: Field });
-export const toMoment = (value: string): Moment => moment(value);
-export const toString = (value: Moment): string => value.toISOString();
+export const toMoment = (value?: string): Moment | undefined => (value ? moment(value) : undefined);
```

Comment:
Unlike in MWs, here we have some nullable dates (i.e. `until`). Without this change the form would display `Invalid date` with empty dates